_find-duplicate-review-by-topic.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+
+import os
+import re
+import sys
+import subprocess
+import textwrap
+
+from typing import List, Optional, Dict, Tuple
+
+# Try to fetch and checkout the review by topic.  The end branch is the topic
```

Comment:
I think it doesn't do that, i.e. it doesn't "fetch and checkout a review". Instead, it lists reviews and look for duplicates, correct?

Reply:
That is indeed, a copy/paste error.